gunicorn/glogging.py

```diff
@@ -195,8 +198,16 @@ def setup(self, cfg):
         self.access_log.setLevel(logging.INFO)
 
         # set gunicorn.error handler
+        if self.cfg.capture_stdio and cfg.errorlog != "-":
```

Comment:
I don't think so. if they pass `-`  then the --capture-stdio is redundant but afaik STDIO will be captured, won't it be?